src/main_screen_play.zig

```diff
@@ -173,6 +173,18 @@ pub fn main() !void {
         try conn.send(&msg);
     }
 
+    // Try to make this window on the bottom (below the main `aim_analyzer` in the
+    // tests).
+    {
+        var msg: [x.configure_window.max_len]u8 = undefined;
+        const len = x.configure_window.serialize(&msg, .{
+            .window_id = ids.window,
+        }, .{
+            .stack_mode = .below,
+        });
+        try conn.send(msg[0..len]);
+    }
```

Comment:
This works in the test environment where there is just the two windows (`screen_play` and the main `aim_analyzer`) but the `.below` is a bit more aggressive than necessary. Ideally, we'd want to use `.sibling = <aim_analyzer_window_id>, .stack_mode = .below,` so it goes just below the `aim_analyzer` window instead of below everything on the desktop.

Reply:
To find the sibling window ID, we'd have to use a `x.query_tree` request and iterate over the windows doing `x.get_property` until we find a matching window.

For reference, the `xwininfo` command ([*source*](https://gitlab.freedesktop.org/xorg/app/xwininfo/-/blob/ab427b9bfd495a7e9a5dc690f5abe5a2859b8de3/clientwin.c)) does this to find a window with a certain name.

Reply:
Updated to do exactly this :white_check_mark:

We `x.query_tree` and then look for a window with the property `madlittlemods.app_id` (`STRING`) = `aim_analyzer`